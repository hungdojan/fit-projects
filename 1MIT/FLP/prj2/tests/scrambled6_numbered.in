462
255
155
533 443 541 532
111 226 131 544
336 436 126 322
661
664
452
