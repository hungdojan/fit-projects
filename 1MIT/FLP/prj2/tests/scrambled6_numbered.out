462
255
155
533 443 541 532
111 226 131 544
336 436 126 322
661
664
452

355
255
155
533 444 116 432
111 226 432 544
336 432 511 222
661
664
663

123
555
555
444 116 432 533
111 226 432 544
336 432 511 222
661
664
663

123
555
555
444 116 432 533
111 226 432 544
222 336 432 511
666
666
341

555
555
555
444 111 222 333
111 222 333 444
222 333 444 111
666
666
666

555
555
555
111 222 333 444
111 222 333 444
222 333 444 111
666
666
666

555
555
555
111 222 333 444
111 222 333 444
111 222 333 444
666
666
666
